nes_bnf.html

```diff
@@ -0,0 +1,1206 @@
+                rozwinDivObj = child;
+                break;
+            }
+        }
+        if ( rozwinDivObj ) {
+            var store = rozwinDivObj.innerHTML;
+            rozwinDivObj.innerHTML = '';
+            html = elem.innerHTML;
+            rozwinDivObj.innerHTML = store;
+        }
+        else {
+            html = elem.innerHTML;
+        }
+        return html;
+    }
+
+    function przywolaneUsingDefHtml ( elems ) {
+        var html = '<div class="uzywajaceDef">';
+        var max=elems.length;
+        if ( max ) {
+            for (var i=0; i < max; i++)
+                html += "\n" + getDefHtml(elems[i]);
+        }
+        else {
+            html += 'NIE JEST UŻYTY W INNYCH DEFINICJACH';
+        }
+        html += "\n" + '<br/><button onClick="closePrzywolanaDef(this)">Schowaj</button></div>';
+        return html;
+    }
+
+    function showDef(elem, status=true) {
+        var id = elem.innerText;
+        var rozwin_div = getExpandDiv(elem);
+        if ( rozwin_div ) {
+            rozwin_div.innerHTML = '';
+            var target_definition_elem = getProperDefObj(id);
+            if  ( target_definition_elem )
+                rozwin_div.innerHTML = przywolanaDefHtml(target_definition_elem);
+        }
+    }
+
+    function showUse(elem, status=true) {
+        var name = elem.innerText;
+        var rozwin_div = getExpandDiv(elem);
+        if ( rozwin_div ) {
+            rozwin_div.innerHTML = '';
+            var definitions = getUsingDefObjs(name);
+            if  ( definitions )
+                rozwin_div.innerHTML = przywolaneUsingDefHtml(definitions);
+        }
+    }
+
+    function turnOnOffXML(on_off) {
+        var tagvalue = on_off ? 'inline' : 'none';
+        var pktag = [ 'ptag', 'ktag', 'pptag', 'pktag'];
+        pktag.forEach( function(tag) {
+            var all = document.getElementsByTagName(tag);
+            for (var i=0, max=all.length; i < max; i++)
+                all[i].setAttribute('style','display:'+tagvalue);
+        })
+        var rawvalue = on_off ? 'none' : 'inline';
+        var rawtags = [ 'attrib' ];
+        rawtags.forEach( function(tag) {
+            var all = document.getElementsByTagName(tag);
+            for (var i=0, max=all.length; i < max; i++)
+                all[i].setAttribute('style','display:'+rawvalue);
+        })
+    }
+
+    function turnAccordinglyXML() {
+        var elem = document.getElementById('ShowXMLOnOff');
+        turnOnOffXML(elem.checked);
+    }
+
+    function turnOnOffHelp(on_off) {
+        // true - pokaz
+        // false - ukryj
+        // undefiined - trigger
+        var helpwnd = document.getElementById('HelpWindow');
+        if ( on_off === undefined )
+            on_off = helpwnd.style.display == 'none';
+        var tagvalue = on_off ? '' : 'none';
+        helpwnd.setAttribute('style','display:'+tagvalue);
+    }
+
+
+</script>
+
+
+</head>
+
+<body>
+
+
+<div class="all">
+<style>

```

Comment:
Jakie jest uzasadnienie CSSa w tym samym pliku?

Jest tutaj też sporo powtórzonego kodu, np. font-family powinien być globalny, a np. `definicja` i `uzycie` mają te same własności CSS, więc nie trzeba powtarzać kodu. Przydałoby się nieco refaktoryzacji, najlepiej zrobić pairing na to.